entries/u/united.com.json

```diff
@@ -2,12 +2,20 @@
   "United": {
     "domain": "united.com",
     "url": "https://www.united.com/",
-    "contact": {
-      "facebook": "United",
-      "twitter": "united"
-    },
-    "categories": [
+    "tfa": [
+      "email",
+      "sms",
+      "call",
+      "totp",
+      "custom-software"
+    ],
+     "custom-software": [
+      "United App"
+    ],
+    "documentation":"https://www.united.com/en/us/fly/mileageplus/protect-your-account.html",
```

Comment:
```suggestion
```

This URL does not qualify as documentation. The documentation URL should contain instructions on how to set up MFA. For example, see [GitHub's documentation page](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication).